webodm.sh

```diff
@@ -171,7 +171,7 @@ detect_gpus(){
 
 	if [ "${platform}" = "Linux" ]; then
 		set +e
-		lspci | grep 'VGA.*NVIDIA'
+		lspci | grep 'NVIDIA'
```

Comment:
In my notebook, it has 2 VGA, Intel UHD and Nvidia MX150.
If I run it in terminal :
```
>>> lspci | grep 'VGA.*Intel'
00:02.0 VGA compatible controller: Intel Corporation UHD Graphics 620 (rev 07)
```

Unfortunatelly it can't detect Nvidia
```
>>> lspci | grep 'VGA.*NVIDIA'
((nothing appear))
```
It works when I did this : 
```
>>> lspci | grep 'NVIDIA'
01:00.0 3D controller: NVIDIA Corporation GP108M [GeForce MX150] (rev a1)
```